src/scripts/main.js

```diff
@@ -1,29 +1,62 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const bodyElem = document.querySelector('body');
+  const message = document.createElement('div');
+
+  message.classList.add('notification');
+
+  message.style.right = posRight + 'px';
+  message.style.top = posTop + 'px';
+
+  const messTitle = document.createElement('h2');
+
+  messTitle.classList.add('title');
+  messTitle.innerHTML = title;
+
+  const messText = document.createElement('p');
+
+  messText.innerHTML = description;
+
+  if (type === 'warning') {
+    message.classList.add('warning');
+  }
+
+  if (type === 'success') {
+    message.classList.add('success');
+  }
+
+  if (type === 'error') {
+    message.classList.add('error');
+  }
+
+  bodyElem.append(message);
+  message.append(messTitle);
+  message.append(messText);
+
+  setTimeout(() => message.remove(), 2000);
```

Comment:
Don't need to remove element, you need just to hide, you can use display or visibility